test/test_http11.rb

```diff
@@ -183,4 +183,30 @@ def test_horrible_queries
     end
 
   end
+
+  def test_13632_workaround
```

Comment:
No, I don't. The `pending_interrupt?` is not expected to be true for all rubies that are not addressed specifically. We'd better see immediately that it started to happen in this case.